source/_integrations/atome.markdown

```diff
@@ -24,7 +24,9 @@ As there is no official documentation for the API, the component retrieves data
 To use it, you need to order the device directly from "Total Direct Energie" Mobile App. Then you need to follow up the installation (covered in the Atome App).
 The configuration (see below) needs your Atome username & password you created during the initialization of the Atome device.
 
-Next, add the Atome sensor to your `configuration.yaml` file like below:
+Next use config flow.
```

Comment:
Use the include we have available for this. Take a glance at any other documentation page using a config flow.